src/app/modules/main/chat_section/module.nim

```diff
@@ -763,7 +773,7 @@ method onCommunityTokenPermissionDeleted*(self: Module, communityId: string, per
   singletonInstance.globalEvents.showCommunityTokenPermissionDeletedNotification(communityId, "Community permission deleted", "A token permission has been removed")
 
 method onCommunityTokenPermissionCreated*(self: Module, communityId: string, tokenPermission: CommunityTokenPermissionDto) =
-  let community = self.controller.getCommunityById(communityId)
+  let community = self.controller.getMyCommunity()
```

Comment:
I don't understand some of the changes here, just find it weird that we're not using the function's `communityId` parameter

Reply:
Yes it's intentional. The passed `communityId` is pretty much useless. I'll try to refactor in a later PR.

The reason why is that this is the chat_section module (the other modules are part of a chat_section). By definition, a chat section is either a community, or the personal chat section (groups and 1-1). That means that passing the communityId is useless, because a chat section only bothers with their own events, so any call to `getCommunityById` was actually to get the DTO for itself, hence why I changed to use `getMyCommunity`.